src/radical/pilot/session.py

```diff
@@ -645,10 +658,10 @@ def get_resource_config(self, resource, schema=None):
         Returns a dictionary of the requested resource config
         '''
 
-        if  resource in self._rcfgs.get('aliases', {}):
-            self._log.warning("using alias '%s' for deprecated resource '%s'"
-                              % (self._rcfgs.aliases.resource, resource))
-            resource = self._rcfgs.aliases.resource
+        if resource in self._rcfgs.aliases.get('aliases', {}):
+            self._log.warning("Using alias '%s' for deprecated resource '%s'" %
+                              (self._rcfgs.aliases.aliases[resource], resource))
+            resource = self._rcfgs.aliases.aliases[resource]
```

Comment:
Why is that `aliases.aliases`, did I mess this up?

Reply:
[File with aliases](https://github.com/radical-cybertools/radical.pilot/blob/1445cfb7a05ebac846ef3c1d39681e46d7a67ed4/src/radical/pilot/configs/resource_aliases.json) has the same approach (`<domain>.<host>`) as regular resource configs. 

p.s. I personally don't see an advantage of using aliases here, since names had already transformed into format as `<machine_name>_<highlighted_feature>`. Maybe in the future, we can merge configs for one machine, but with different launch methods (and/or other features), the same way as it is done for access `schemas`, thus would extend inputs for `get_resource_config` as following: machine name, schema and launch method. Just general not much-related comments :)

Reply:
I agree, we probably should get rid of the aliases.  They were introduced to keep backward compatibility after a cfg name change -- that was years ago...  Lets remove them/...

Reply:
should I remove them in this PR? (they are just in session module, so it would be straightforward; and will remove the corresponding file from `.../configs/`)

Reply:
Yes, makes sense - please do.